lib/service_registry.rb

```diff
@@ -0,0 +1,17 @@
+class ServiceRegistry
```

Comment:
The service registry implementation differs slightly from the one in Rota, do you think it would be a good idea to keep them the same?